src/server/worldserver/Main.cpp

```diff
@@ -46,14 +46,14 @@ uint32 realmID;                                             ///< Id of the realm
 /// Print out the usage string for this program on the console.
 void usage(const char* prog)
 {
-    printf("Usage:\n");
-    printf(" %s [<options>]\n", prog);
-    printf("    -c config_file           use config_file as configuration file\n");
+    sLog->outString("Usage:\n"
+					" %s [<options>]\n"
+					"    -c config_file           use config_file as configuration file\n\r", prog);
```

Comment:
I didn't mean to change from `printf` to `sLog->outString` where it already was `printf` (however we can keep this change if it works fine), I just meant to NOT change existing `sLog` with `printf` because of no reason :)

Reply:
ok thanks